Source/Charts/Charts/BarLineChartViewBase.swift

```diff
@@ -57,19 +57,28 @@ open class BarLineChartViewBase: ChartViewBase, BarLineScatterCandleBubbleChartD
     /// **default**: false
     @objc open var keepPositionOnRotation: Bool = false
     
-    /// the object representing the left y-axis
-    internal var _leftAxis: YAxis!
+    /// The left y-axis object. In the horizontal bar-chart, this is the
+    /// top axis.
+    @objc open internal(set) var leftAxis: YAxis!
```

Comment:
what's the `set` purpose here?

Reply:
Previously there were two variables
```
internal var _leftAxis: YAxis!
@objc open var leftAxis: YAxis! {
    return _leftAxis
}
```

There is no need for this as swift provides better access control than objective-c did previously. `open internal(set)` (can also be `open private(set)` but was holding off on that one until I evaluated whether that was appropriate) makes the variable `open`, but only allows it to be set by internally.

Reply:
wow, I don't realize swift add this already.

Reply:
Just a moment. Have you checked if people can set axis outside from the framework? Just like other PRs similar issue?

Reply:
I just did a quick check, axis is not settable outside, seems safe. Just run the tests and demo:)